plugins/generator-1.20.4/neoforge-1.20.4/templates/livingentity/livingentity.java.ftl

```diff
@@ -216,18 +208,18 @@ public class ${name}Entity extends ${extendsClass} <#if data.ranged>implements R
     </#if>
 
 	<#if data.mobModelName == "Biped">
-	@Override public double getMyRidingOffset() {
-		return -0.35D;
+	@Override protected float ridingOffset(Entity entity) {
+		return -0.35F;
 	}
 	<#elseif data.mobModelName == "Silverfish">
-	@Override public double getMyRidingOffset() {
-		return 0.1D;
+	@Override protected float ridingOffset(Entity entity) {
+		return 0.1F;
 	}
 	</#if>
 
 	<#if data.mountedYOffset != 0>
-	@Override public double getPassengersRidingOffset() {
-		return super.getPassengersRidingOffset() + ${data.mountedYOffset};
+	@Override public Vec3 getPassengerRidingPosition(Entity entity) {

```

Comment:
Isn't getPassengerAttachmentPoint more suitable? All vanilla entities use getPassengerAttachmentPoint and not getPassengerRidingPosition which also covers orientation

Reply:
I only noticed this method :man_shrugging: